### PR TITLE
03 bash 1

### PR DESCRIPTION
**Pull request by prosolvo4th**
find_dup_files.sh

### CODE REVIEW

File: 03-Bash/find_dup_files/find_dup_files.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#!/bin/bash +x
```
Comment (yekovalyov) [inline 03-Bash/find_dup_files/find_dup_files.sh]: Is this option (-x) required in release version?

File: 03-Bash/find_dup_files/find_dup_files.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+#create file with checksum for each file
+cat fileList | while read line
+do
+    tmp=$(md5sum "$line") 
+    echo $tmp | cut -d ' ' -f1 > "$line".md5sum
+done
+
+# compare contents of *.md5sum files
+cat fileList | while read md5File1
```
Comment (yekovalyov) [inline 03-Bash/find_dup_files/find_dup_files.sh]: The complexity of this algorithm is O^2. Please consider another way for finding duplicate files.

Comment (prosolvo4th) [reply 03-Bash/find_dup_files/find_dup_files.sh]: Do you mean the md5sum comparison mechanism?

File: 03-Bash/find_dup_files/find_dup_files.sh (hunk 1 above)
Comment (prosolvo4th) [reply 03-Bash/find_dup_files/find_dup_files.sh]: It's mistake.

File: 03-Bash/find_dup_files/find_dup_files.sh (hunk 2 above)
Comment (yekovalyov) [reply 03-Bash/find_dup_files/find_dup_files.sh]: Yes. You compare each of N records with other. But it is possible to sort all records and compare only the neighbors.

**Pull request merged**